src/fragments/lib/auth/common/mfa/flows.mdx

```diff
@@ -0,0 +1,318 @@
+## SMS flows
```

Comment:
for the intro of this section - it should just state that in order to secure your app, you can add MFA. And that when you add MFA, you have two options to do that, whether through SMS or Time-based one-time passwords. All this info about how phone numbers are used are unnecessary.

Reply:
Done.